MediaBrowser.Controller/MediaEncoding/JobLogger.cs

```diff
@@ -39,6 +39,13 @@ public async void StartStreamingLog(EncodingJobInfo state, Stream source, Stream
                         }
 
                         await target.WriteAsync(bytes, 0, bytes.Length).ConfigureAwait(false);
+
+                        // Check again, the stream could have been closed
+                        if (!target.CanWrite)
```

Comment:
This feels like a incomplete fix, as `target.CanWrite` can become false right after you checked it... with this check you're just lowering the probability of the issue happening here.
It feels like some synchronization is missing somewhere in the architecture...

Reply:
@JustAMan But is it good enough for now? Or should we not bother. I'm not sure what issue(s) this fixes.

Reply:
I dislike solutions like this, because what they usually do is they lower the probability of an exception to a level when it could still happen to annoy users but is almost impossible to catch or reproduce in development...

That said, if others would approve go ahead and merge.

Reply:
The impact is minimal though. The exception is caught and logged. Nothing more.

Reply:
@cvium feel free to approve and merge :)